src/visu_config.xsd

```diff
@@ -598,12 +598,42 @@
             <xsd:appinfo>level:expert</xsd:appinfo>
         </xsd:annotation>
     </xsd:attribute>
-    <xsd:attribute name="showtopnavigation" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="showfooter" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-top" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-bottom" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-left" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-right" type="xsd:boolean" use="optional" />
+    <xsd:attribute name="showtopnavigation" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">the navigation path shows on which subpage of the visu one currently is</xsd:documentation>
+            <xsd:documentation xml:lang="en">Der Navigationspfad zeigt an, auf welcher Seite der Visualisierung man sich befindet</xsd:documentation>
+        </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="showfooter" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">Shows the CometVisu logo and various Links</xsd:documentation>
+            <xsd:documentation xml:lang="en">Zeigt die Fußzeile mit CometVisu Logo und diversen Links an</xsd:documentation>
```

Comment:
The CometVisu logo and the links are just one way to use the footer. The content is set in the config and thus can show many different this